Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/TelementryDiagnosticControlsImpl.java

```diff
@@ -0,0 +1,29 @@
+package tddmicroexercises.telemetrysystem;
+
+public class TelementryDiagnosticControlsImpl implements TelemetryDiagnosticInterface {
```

Comment:
Lets not suffix or prefix with Interface and Impl
We can have TelementryDiagnosticControls for Implementation and DiagnosticControls for Interface